app/views/course/forum/posts/_post.html.slim

```diff
@@ -0,0 +1,27 @@
+= content_tag_for(:post, post) do
+  div.post id="post-#{post.id}"
+    div.user
+      h5 = link_to_user(post.creator)
+
+    div.contents
+      div.pull-right
+        .timestamp
+          => format_datetime(post.created_at)
+
+      h4.title
+        a href="#post-#{post.id}"
```

Comment:
You should use content tag for, and change all the IDs to use the content_tag_for convention